### PR TITLE
new introspection

### PR DESCRIPTION
**Pull request by vogievetsky**
let's squash those introspection issues.

### CODE REVIEW

**Comment by vogievetsky** [comment]
Please do not merge yet, I want to get a sign off from @gianm

File: test/external/druidExternalIntrospection.mocha.coffee [hunk 1]
```diff
@@ -0,0 +1,525 @@
...
+      expect(query.merge).to.equal(true)
+      expect(query.analysisTypes).to.be.an('array')
+
+      merged = {
+        "id": "merged",
+        "intervals": null,
+        "size": 0,
+        "numRows": 654321,
+        "columns": {
+          "__time": { "type": "LONG", "hasMultipleValues": false, "size": 0, "cardinality": null, "errorMessage": null },
```
Comment (gianm) [inline test/external/druidExternalIntrospection.mocha.coffee]: 0.8.3 does not have `hasMultipleValues` although this is probably not affecting the usefulness of the test.

**Comment by gianm** [comment]
:+1:

File: test/external/druidExternalIntrospection.mocha.coffee (hunk 1 above)
Comment (vogievetsky) [inline test/external/druidExternalIntrospection.mocha.coffee]: Yep I am aware, it should really be requesterDruid_0_8_3_IAP but that is just too long. The `hasMultipleValues` is really a side player here really this is testing 'aggregate' and introspection fallbacks.

**Comment by evasomething** [comment]
:droplet:

**Pull request merged**